### ISSUE TITLE
VR offsets with commonlibvr

### ISSUE THREAD

**Issue by alandtse** [Seally/yastm#2]
Just wanted to give you a heads-up that if you use my version of commonlibvr, you don't have to necessarily define VR offsets inline like you do here: https://github.com/Seally/yastm/blob/main/src/offsets.hpp#L24

Instead, if you submit the offset to me at [vr_address_tools](https://github.com/alandtse/vr_address_tools), then you can continue to use the SSE address ID. Of course, I don't have a solution for the in-function [offsets](https://github.com/Seally/yastm/blob/main/src/offsets.hpp#L116), so you'll have to have code like you currently have. You can see everything I've already discovered in this [file](https://github.com/alandtse/vr_address_tools/blob/main/database.csv).

I do have tools to try to basically lint for undefined addresses in my database, but i don't currently support your macro format. https://github.com/alandtse/vr_address_tools/issues/5, but I've opened an issue to add it at some point.

**Comment by Seally** [comment]
The issue is that I didn't have a clue at the time if the offsets even worked. I actually used your mappings to create the VR port since I have no access whatsoever to Skyrim VR, so when I made it it was basically just hoping those mappings alone would be enough. 

I based the function offsets on ones I used for SE v1.5, hoping that Bethesda used the same compiler and didn't touch anything in the patched functions. It took over a month for my "tester" to get back to me and verify it all mostly works.
